BuildTools/Mintfile

```diff
@@ -1,2 +1,2 @@
-nicklockwood/SwiftFormat@0.49.6
-realm/SwiftLint@0.47.0
\ No newline at end of file
+nicklockwood/SwiftFormat
+realm/SwiftLint
```

Comment:
I'm not sure about this. While it's nice to keep everything updated, this is the only thing that keeps all our machines and CI running the same version - and eventually this could introduce breaking changes.

I'm curious though, were you running it through Mint?